src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -31,6 +31,7 @@ interface IBackupMetaData {
 	size: number;
 	etag: string;
 	orphaned: boolean;
+	readonly?: boolean;
```

Comment:
I would not add `readonly` to backups as I see no benefit doing so.

Reply:
Removed in latest push.